mlflow/spacy.py

```diff
@@ -295,3 +299,114 @@ def load_model(model_uri):
     # `data` key; in this case, we assume the model artifact path to be `model.spacy`
     spacy_model_file_path = os.path.join(local_model_path, flavor_conf.get("data", "model.spacy"))
     return _load_model(path=spacy_model_file_path)
+
+
+@experimental
+@autologging_integration(FLAVOR_NAME)
+def auto_logger(
```

Comment:
https://github.com/mlflow/mlflow/blob/0f9d1e04975c69313ad57e4d7d428ab7befad69e/mlflow/tracking/fluent.py#L1455-L1460 seems like the convention is to call it "autolog"

Reply:
I'll try to adapt for this. Thanks!